sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3811,6 +3811,8 @@ class SQLConf extends Serializable with Logging {
    * definition contains key, defaultValue and doc.
    */
   def getAllDefinedConfs: Seq[(String, String, String, String)] = sqlConfEntries.synchronized {
+    // Eagerly load conf objects to force the conf entries registering
+    SparkConfRegisterLoader.load(Utils.getContextOrSparkClassLoader)
```

Comment:
shall we make sure this is called globally once?

Reply:
sure, will do